plugins/MUN Identity/__init__.py

```diff
@@ -210,11 +212,39 @@ def check(reaction, user):
             return user == ctx.author and str(reaction.emoji) in ["✅", "❌"]
 
         try:
-            reaction, _ = await ctx.bot.wait_for("reaction_add", check=check, timeout=30)
+            reaction, _ = await ctx.bot.wait_for(
+                "reaction_add", check=check, timeout=30
+            )
         except asyncio.TimeoutError:
             return False
         else:
             if reaction.emoji == "✅":
                 return True
             elif reaction.emoji == "❌":
                 return False
+
+    @identity.command(name="restore_roles")
+    @commands.has_permissions(view_audit_log=True)
+    async def identity_restore_roles(self, ctx: commands.Context):
+        """Restores VERIFIED_ROLE to users with a registered identity who were not granted it."""
+        members_restored: List[nextcord.Member] = []
+        async with ctx.typing():
+            for identity in self.identities.find():
+                member: nextcord.Member = self.bot.get_guild(PRIMARY_GUILD).get_member(
+                    identity["discord_id"]
+                )
+                if member is None or VERIFIED_ROLE in member.roles:
+                    continue
+                await member.add_roles(
+                    self.bot.get_guild(PRIMARY_GUILD).get_role(VERIFIED_ROLE),
+                    reason=f"Identity restored by {ctx.author.name}#{ctx.author.discriminator}.",
+                )
+                members_restored.append(member)
+                await asyncio.sleep(0.1)
```

Comment:
Might want to sleep for a bit more so we don't get rate limited

Reply:
Makes sense, well it's no rush to restore. Think bumping it to 1s is fine?